tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -926,13 +926,25 @@ public void exec(int depth, int concurrencyDepth, AlluxioURI path) throws Except
     }
   }
 
+  /**
+   * This class provides multiple concurrent threads to rename all files in one directory.
+   */
   class ConcurrentRenamer implements Callable<Void> {
     private int mDepth;
     private int mConcurrencyDepth;
     private AlluxioURI mRootPath;
     private AlluxioURI mRootPath2;
     private AlluxioURI mInitPath;
 
+    /**
+     * Constructs the concurrent renamer.
+     *
+     * @param depth the depth of files to be renamed in one directory
+     * @param concurrencyDepth the concurrency depth of files to be renamed in one directory
```

Comment:
Sometimes we want to try renaming a path when we're not all the way down, which is what
the concurrency depth for, but may be it is a little difficult to explain clearly to readers without knowing about the source  code.